specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2019-09-01-preview/QueryPackQueries_API.json

```diff
@@ -0,0 +1,477 @@
+        },
+        "body": {
+          "type": "string",
+          "readOnly": false,
+          "description": "Body of the query."
+        },
+        "linkedResourceId": {
+          "type": "string",
+          "readOnly": false,
+          "description": "Resource id associated with the query."
+        },
+        "categories": {
+          "description": "Categories associated with the query.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "resourceTypes": {
+          "description": "Resource Types associated with the query.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "labels": {
+          "description": "Labels associated with the query.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      },
+      "required": ["body", "displayName"]
+    },
+    "LogAnalyticsQueryPackQuerySearchProperties": {
+      "description": "Properties that define an Log Analytics QueryPack-Query search properties.",
+      "properties": {
+        "categories": {
+          "description": "Categories associated with the query.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "resourceTypes": {
+          "description": "Labels associated with the query.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "labels": {
+          "description": "Resource Types associated with the query.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "LogAnalyticsQueryPackQueryListResult": {
+      "description": "Describes the list of Log Analytics QueryPack-Query resources.",
+      "required": ["value"],
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of Log Analytics QueryPack Query definitions.",
+          "items": {
+            "$ref": "#/definitions/LogAnalyticsQueryPackQuery"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URI to get the next set of Log Analytics QueryPack definitions if too many QueryPack-Queries where returned in the result set."
+        }
+      }
+    }
+  },
+  "parameters": {
+    "QueryPackNameParameter": {
+      "name": "queryPackName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The name of the Log Analytics QueryPack resource.",
+      "x-ms-parameter-location": "method"
+    },
+    "QueryIdParameter": {
+      "name": "queryId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The id of a specific query defined in the Log Analytics QueryPack",
+      "x-ms-parameter-location": "method"
+    },
+    "IncludeBodyParameter": {
```

Comment:
>"IncludeBodyParameter" [](start = 4, length = 22)

The default if this parameter is not included is to include the body, correct?

Reply:
exactly